uaclient/log.py

```diff
@@ -0,0 +1,59 @@
+import json
+import logging
+import re
+from collections import OrderedDict
+from typing import Any, Dict, List
+
+
+class JsonArraryFormatter(logging.Formatter):
+    default_time_format = "%Y-%m-%dT%H:%M:%S%z"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if not self.datefmt:
+            self.datefmt = self.default_time_format
```

Comment:
Does this need to happen in the constructor?
We can just include this logic in-line in the `format` function.
Or we can just hardcode the date format since we don't have to support a configuration for it